src/analysis.js

```diff
@@ -35,7 +35,7 @@ function addProxyAgent(options, opts) {
  * @param {string} url - the backend url to send the request to
  * @param {boolean} [html=false] - true will return 'text/html', false will return 'application/json'
  * @param {{}} [opts={}] - optional various options to pass along the application
- * @returns {Promise<string|import('../generated/backend/AnalysisReport').AnalysisReport>}
+ * @returns {Promise<string|import('AnalysisReport').AnalysisReport>}
```

Comment:
Does this work? I wouldve thought it should be:
```suggestion
 * @returns {Promise<string|import('@trustification/exhort-api-spec').AnalysisReport>}
```

Reply:
You're right, thanks!